tools/pages/admin/permit-holders/permit-holder-id.ts

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
I think this file could be better named - I wouldn't be sure what `permit-holder-id` means as a developer who would be new to the codebase. `types.ts` would be better.

Also please add some comments to the types in this file so that we know where they're used